src/components/datepicker/DatePickerInput.tsx

```diff
@@ -0,0 +1,197 @@
+import React, { useEffect, useState } from 'react';
+import Box, { pickBoxProps } from '../box';
+import DatePicker from '.';
+import Icon from '../icon';
+import Input from '../input';
+import Popover from '../popover';
+import cx from 'classnames';
+import theme from './theme.css';
+import { formatDate } from './localeUtils';
+import { IconCalendarSmallOutline } from '@teamleader/ui-icons';
+import { DateTimeFormatOptions } from 'luxon';
+import { InputProps } from 'react-select';
+
+interface DatePickerInputProps {
```

Comment:
Needs to extend from BoxProps.
All of the props are now required as well. Pretty sure they are not.